packages/components/app/styles/components/app-footer.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// app-footer
+//
+
+// WIP: names, etc. may change
+// :root {
+//   --token-app-footer-text-color: var(--token-color-foreground-faint);
+//   --token-app-footer-border-color: var(--token-color-border-primary);
+// }
+
+// NOTE: Styles are a WIP and NOT final
+.hds-app-footer {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 24px;
+  justify-content: flex-end;
+
+  padding: 24px;
+  border-top: 1px solid var(--token-color-border-primary);
+}
+
+.hds-app-footer__list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 24px;
+  align-items: center;
+  justify-content: flex-end;
+  width: max-content; // testing
+  margin: 0;
+  padding: 0;
+}
+
+.hds-app-footer__list-item { list-style-type: none; }
+
+.hds-app-footer__link {
+  // Overriding some default colors
+  &.hds-link-inline--color-secondary {
+    color: var(--token-color-foreground-faint);
+
+    &:hover,
+    &.mock-hover {
+      color: var(--token-color-palette-neutral-600);
+    }
+
+    &:active,
+    &.mock-active {
+      color: var(--token-color-palette-neutral-700);
+    }
+
+    &:focus,
+    &.mock-focus {
+      color: var(--token-color-foreground-faint);
+    }
+  }
+}
+
+.hds-app-footer__link,
```

Comment:
The `.hds-app-footer__link` is an `<Hds::Link::Inline>` which is already a flex, with its own gap + alignments, so no need to have this. 

Which leads me to another point: what if this flex styling is applied to the `hds-app-footer__list-item` directly, as general rule (so it works even if consumers decide to add a custom item with some content in it (and if they don't want the flaxbox, they can always wrap the content in a `<div>`)
In this way, also, you can get rid of the `hds-app-footer__list-item--copyright` qualifier

Reply:
Thanks, some of the styles are left over from before I changed it to use Link::Inline. I will clean this up. The styles still are still not really ready for review. I plan to clean them up today as part of implementing the more finalized visual designs from @andgen404

Reply:
This was partially updated after I pushed up refactors for the StatusLink. I'm updating to also apply the layout to the list-item as you suggested, thanks. (I originally wasn't sure of how much custom styling I should add directly to the list-item since it can accept custom content.)

Reply:
I believe I've resolved this.